Automattic/wp-calypso
# Add/account recovery/step 5b to 6

## Summary
This PR is based on #13360 and chains the step 5b to the step 6:
![demo-ar-5-6](https://cloud.githubusercontent.com/assets/1842898/25371722/a67dcb3e-29c4-11e7-9d74-6cf483a58a70.gif)

## Test Plan
1. Open an incognito window and make sure you are logged out.
1. Visit http://calypso.localhost:3000/account-recovery
1. Use a test account info to proceed with instructions, but make sure to choose "reset by the primary sms number" or "reset by the recovery sms number".
1. Enter the received code, and you should arrive the final "New password" form.

<strike>Note that for the final test step, if D5299 is not deployed, you will be directed to the next form no matter what you've entered. Also, </strike>the final "New password" form doesn't do anything for now.
